build.sbt

```diff
@@ -52,96 +91,44 @@ lazy val sonatypeSettings = Seq(
-val scalacheckVersion     = "1.14.0"
-
-// Dependencies
-val cats             = "org.typelevel"               %% "cats-core"           % CatsVersion
-val circeCore        = "io.circe"                    %% "circe-core"          % CirceVersion
-val circeFs2         = "io.circe"                    %% "circe-fs2"           % CirceFs2Version
-val circeGeneric     = "io.circe"                    %% "circe-generic"       % CirceVersion
-val circeParser      = "io.circe"                    %% "circe-parser"        % CirceVersion
-val circeRefined     = "io.circe"                    %% "circe-refined"       % CirceVersion
-val circeShapes      = "io.circe"                    %% "circe-shapes"        % CirceVersion
-val geotrellisVector = "org.locationtech.geotrellis" %% "geotrellis-vector"   % GeoTrellisVersion
-val refined          = "eu.timepit"                  %% "refined"             % RefinedVersion
-val scalacheck       = "org.scalacheck"              %% "scalacheck"          % scalacheckVersion % Test
-val scalacheckCats   = "io.chrisdavenport"           %% "cats-scalacheck"     % scalacheckCatsVersion % Test
-val scalatest        = "org.scalatest"               %% "scalatest"           % scalatestVersion % Test
-val spdxChecker      = "com.github.tbouron"          % "spdx-license-checker" % spdxCheckerVersion
-val shapeless        = "com.chuusai"                 %% "shapeless"           % ShapelessVersion
-val spray            = "io.spray"                    %% "spray-json"          % sprayVersion
-
-lazy val settings = Seq(
-  organization := "com.azavea",
-  name := "stac4s",
-  version := "0.0.1-SNAPSHOT",
-  scalaVersion := "2.12.10",
-  scalafmtOnCompile := true,
-  scapegoatVersion in ThisBuild := Versions.ScapegoatVersion,
-  scapegoatDisabledInspections := Seq("ObjectNames", "EmptyCaseClass"),
-  unusedCompileDependenciesFilter -= moduleFilter("com.sksamuel.scapegoat", "scalac-scapegoat-plugin"),
-  addCompilerPlugin("org.spire-math" %% "kind-projector"     % "0.9.6"),
-  addCompilerPlugin("com.olegpy"     %% "better-monadic-for" % "0.2.4"),
-  addCompilerPlugin(
-    "org.scalamacros" % "paradise" % "2.1.0" cross CrossVersion.full
-  ),
-  addCompilerPlugin(scalafixSemanticdb),
-  autoCompilerPlugins := true,
-  externalResolvers := Seq(
-    DefaultMavenRepository,
-    Resolver.sonatypeRepo("snapshots"),
-    Resolver.typesafeIvyRepo("releases"),
-    Resolver.bintrayRepo("azavea", "maven"),
-    Resolver.bintrayRepo("azavea", "geotrellis"),
-    "locationtech-releases" at "https://repo.locationtech.org/content/groups/releases",
-    "locationtech-snapshots" at "https://repo.locationtech.org/content/groups/snapshots",
-    Resolver.bintrayRepo("guizmaii", "maven"),
-    Resolver.bintrayRepo("colisweb", "maven"),
-    "jitpack".at("https://jitpack.io"),
-    Resolver.file("local", file(Path.userHome.absolutePath + "/.ivy2/local"))(
-      Resolver.ivyStylePatterns
-    )
-  )
+lazy val credentialSettings = Seq(
```

Comment:
Some improvements to this block, courtsey of Eugene, https://github.com/geotrellis/vectorpipe/pull/120/commits/227de4ffea27c681dfc08526d37f2fb868fd0c78.